code/modules/antagonists/eldritch_cult/knowledge/void_lore.dm

```diff
@@ -99,7 +99,7 @@
 /datum/eldritch_knowledge/rune_carver
 	name = "Carving Knife"
 	gain_text = "Etched, carved... eternal. I can carve the monolith and evoke their powers!"
-	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them, to create the carving knife transmute a knife with a glass shard and a piece of paper."
+	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them. You can also use them as a good throw weapon, to create the carving knife transmute a knife with a glass shard and a piece of paper."
```

Comment:
throwing != throw, I personally think the current structure is a bit off, but that part's up to you
```suggestion
	desc = "You can create a carving knife, which allows you to create up to 3 carvings on the floor that have various effects on nonbelievers who walk over them. They make quite a handy throwing weapon. To create the carving knife transmute a knife with a glass shard and a piece of paper."
```

Reply:
also use better commit names

Reply:
alright, I'm still quite new to all of this 🙂

Reply:
it's all good, we're all new deep down